### DISCUSSION TITLE
How do you update metals when using emacs?

### DISCUSSION THREAD

**Discussion by kschwarz1116** [scalameta/metals#5887] [Q&A] [answered]
I spent a while determining that in order to update the metals version, you don't just update the lsp-metals package, you also need to run `cs update metals` to get the updated metals server. Would be nice to see this in the docs somewhere (and ideally have the package do this automatically for you), but maybe I'm not understanding something.

**Comment by tgodzik** [comment, marked as answer]
I don't think there is currently a way to do it automatically, though to be sure it would probably be better to ask in the metals-lsp repository https://github.com/emacs-lsp/lsp-metals
